plugins/SitesManager/GtmSiteTypeGuesser.php

```diff
@@ -66,17 +67,84 @@ public function guessSiteTypeFromResponse($response)
         return SitesManager::SITE_TYPE_UNKNOWN;
     }
 
+    /**
+     * Detect GA3 usage from the site data
+     *
+     * @param array $response Extended HTTP Response
+     * @return bool
+     */
+    public function detectGA3FromResponse(array $response): bool
+    {
+        if (empty($response['data'])) {
+            return false;
+        }
+
+        if (strpos($response['data'], '(i,s,o,g,r,a,m)') !== false) {
+            return true;
+        }
+
+        $tests = [
+            "/UA-\d{5,}-\d{1,}/", "/google\-analytics\.com\/analytics\.js/", "/window\.ga\s?=\s?window\.ga/",
+            "/google[ _\-]{0,1}analytics/i"
```

Comment:
Isn't this detection a bit to general? As soon as a page contains `google analytics` it will now be detected as GA3.
This can be seen in the failing UI tests. The Updater test uses `piwik.net` as domain, which redirects to `piwik.org` and finally to `matomo.org`. As `matomo.org` contains the headline `Google Analytics alternative...` it is detected as GA3 🙈

Reply:
If we could keep it in there, that be great

Reply:
ok. let's keep it then